### PR TITLE
facebook: update requests after v2.1 API depecation. Fixes #11283

### LINKED ISSUE #11283: [FR] save filter to collections as preset.
**Is your feature request related to a problem? Please describe.**
First of all thanks @Nilvus and everyone else involved for this new addition to dt. The ability to filter all collections using the top bar with all those filter options makes dt more powerful day by day. 

**Describe the solution you'd like**
I really don't know what other features you are planning to incorporate, but I have a suggestion. It would be great to have the opportunity to save (store) these filters as a new preset in the collections presets.
That way it would be faster to select those presets. Of course, I don't know how to create a naming convention for them, maybe the only way is by editing the preset.

**Additional context**
This is a Mockup with a button to save (storage) as preset.
Screenshot-20220306231604-1218x133

In that case if I search for all the photos of laura with the green color label, I can save this filter as a preset in the collections presets.

Screenshot from 2022-03-06 23-45-47

### PR DESCRIPTION
**Pull request by chubinou**
https://redmine.darktable.org/issues/11283

### CODE REVIEW

**Comment by LebedevRI** [comment]
Someone else who does export to fb should test this.

**Comment by LebedevRI** [comment]
I got @patdavid to try it, and it did not work for him.
@chubinou you are sure that it is currently working, and no other changes are needed?

**Comment by patdavid** [comment]
Signed in ok. The message on attempting to export:

dt FB error

File: src/imageio/storage/facebook.c [hunk 1]
```diff
@@ -46,7 +46,7 @@ DT_MODULE(1)
 
 #define FB_CALLBACK_ID "facebook"
 #define FB_WS_BASE_URL "https://www.facebook.com/"
-#define FB_GRAPH_BASE_URL "https://graph.facebook.com/"
+#define FB_GRAPH_BASE_URL "https://graph.facebook.com/v2.8/"
```
Comment (LebedevRI) [inline src/imageio/storage/facebook.c]: darktable is unable to get a list of albums.

**Comment by chubinou** [comment]
I don't know why it worked when I tested my patch or if there is some subtleties I didn't see.

**Comment by LebedevRI** [comment]
@chubinou thanks.

**Pull request merged**